applications/mne_analyze/plugins/annotationmanager/annotationmanager.cpp

```diff
@@ -76,8 +76,7 @@ AnnotationManager::~AnnotationManager()
 
 QSharedPointer<AbstractPlugin> AnnotationManager::clone() const
 {
-    QSharedPointer<AnnotationManager> pAnnotationManagerClone(new AnnotationManager);
-    return pAnnotationManagerClone;
+    return QSharedPointer<AnnotationManager> (new AnnotationManager);
```

Comment:
These seem to be changes that fit the Event pr, were these mistakenly pushed here?